app/components/overview-info-form/component.js

```diff
@@ -28,61 +30,69 @@ var generateValidators = function(questions) {
 
 const questions = {
   q1: {
-    question: 'Age',
+    question: translations.survey.sections['1'].questions['1'].label,
```

Comment:
Sorry for not noticing this earlier, but I'm a little concerned about the structure of `translation.survey.sections`. [JS objects are unordered](http://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order), meaning we can't reliably iterate over these values (in a consistent order). 

Can `transations.survey.sections` be an array? Likewise for `sections[X].questions`?